pkgs/development/libraries/libime/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, boost
+, python3
+, fcitx5
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libime";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
```

Comment:
You may use https://download.fcitx-im.org/fcitx5/fcitx5/fcitx5-5.0.1_dict.tar.xz instead of fetchFromGithub. This release tarball bundle a dictionary.